torchvision/transforms/transforms.py

```diff
@@ -1545,3 +1544,62 @@ def forward(self, img):
             x, y, h, w, v = self.get_params(img, scale=self.scale, ratio=self.ratio, value=value)
             return F.erase(img, x, y, h, w, v, self.inplace)
         return img
+
+
+class GaussianBlur(torch.nn.Module):
+    """Blurs image with randomly chosen Gaussian blur.
+    The image can be a PIL Image or a Tensor, in which case it is expected
+    to have [..., 3, H, W] shape, where ... means an arbitrary number of leading
+    dimensions
+
+    Args:
+        rad_min (float): Minimum radius that can be chosen for blurring kernel.
+        rad_max (float): Maximum radius that can be chosen for blurring kernel.
+
+    Returns:
+        PIL Image or Tensor: Gaussian blurred version of the input image.
+
+    """
+
+    def __init__(self, rad_min=0.1, rad_max=2.0):
```

Comment:
I recommend the input to be consistent with those of `ColorJitter`, i.e., `float or tuple of float (min, max)`.
https://github.com/pytorch/vision/blob/a4736ea6e8ff25a1265ca5adf9e6e244d78500e8/torchvision/transforms/transforms.py#L1013-L1019

Reply:
If the `radius` is passed as `float`, does that imply we are going to choose a kernel with that fixed radius every time? Or would it mean we will choose a radius randomly between, (1-radius, 1+radius), like in `ColorJitter`?
The former one seems more intuitive to me because unlike brightness (and others in `ColorJitter`) there is no "central value"(=1 in `ColorJitter`) of radius here which indicates that it is the default one. What do you think?

Reply:
I think a fixed radius would be fine if it is passed as float.